recipes/pdb_manip_py/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pdb_manip_py" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 204d3a121c05075564d2cf08466b6acf5d6c072aab6fa03c926e97f89469ddba
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
```

Comment:
I believe you only need `pip` and `python` under `host` since the others are just run time dependencies

Reply:
Thanks for your comments.

> I believe you only need `pip` and `python` under `host` since the others are just run time dependencies

I'll give it a try. Thanks

> I'm not sure what's going on exactly with the windows failure. I'm not sure if it's because the binary is actually missing or if `os_command_py` is not doing what the test things it should be doing.

The issue is from the os_command_py package (that I coded myself) and the which() function which finds path of executable (as in bash). It looks for all combination within the `$PATH` env variable.
It's probably not the best option. Any suggestions ?